
running 2 tests
test merge::test ... ok
test test ... FAILED

failures:

---- test stdout ----

--- toml config ---
[a]
b=1
c="x"
f=3

--- env with prefix TEST_ ( set by direnv & ./.env ) ---

TEST_server__host=127.0.0.1
TEST_admin=xxai.art@gmail.com
TEST_grpc_port=3222
TEST_site__title="xxAI.Art - 我们计算艺术"

--- convert env into toml ---
grpc_port=3222
admin=xxai.art@gmail.com
[site]
title="xxAI.Art - 我们计算艺术"
[server]
host=127.0.0.1

thread 'test' panicked at 'called `Result::unwrap()` on an `Err` value: Error { inner: Error { inner: TomlError { message: "invalid string\nexpected `\"`, `'`", original: Some("grpc_port=3222\nadmin=xxai.art@gmail.com\n[site]\ntitle=\"xxAI.Art - 我们计算艺术\"\n[server]\nhost=127.0.0.1\n"), keys: [], span: Some(21..22) } } }', src/lib.rs:27:36
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    test

test result: FAILED. 1 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

